library/php-zendserver

```diff
@@ -8,5 +8,8 @@ Directory: 8.5/5.6
 Tags: 9.1
 Directory: 9.1/7.1
 
+Tags: 2018.0
+Directory: 2018.0/7.2
```

Comment:
Is this being added back because it is still actively supported and will be receiving more updates?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
It is not actively supported anymore, but it should be still available. I think it makes sense then not to add back the tag.